### PR TITLE
Add L-KaaS docs

### PR DESCRIPTION
**Pull request by ntnguyencse**
Add L-KaaS docs

### CODE REVIEW

**Review by bactp — APPROVED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Logical Cluster as a Service (L-KaaS) Documentation:
+## L-KaaS Introduction:
+
+* L-KaaS is a project focused on providing declarative APIs and tooling to simplify, abstract, be easy to use for users who don’t have deep technical knowledge of infrastructure and shield them from low-level concepts and technologies.
+
+* Cluster API is a project which is a Kubernetes project to bring declarative, Kubernetes-style APIs to cluster creation, configuration, and management but Cluster API still requires complex configuration, deep knowledge of Kubernetes, cloud provider, relies on several components being configured correctly to have a working cluster. L-KaaS uses Kubernetes Resources Model and Kubernetes environment to provide an abstraction high-level & automation framework that automates clusters, logical clusters. The L-KaaS is a project standing on the Cluster API project.
```
Comment (Jangwon-Lee-DCN) [inline docs/README.md]:
```suggestion
* Cluster API is a Kubernetes subproject to bring declarative, Kubernetes-style APIs to cluster creation, configuration, and management but Cluster API still requires complex configuration, deep knowledge of Kubernetes, cloud provider, relies on several components being configured correctly to have a working cluster. L-KaaS uses Kubernetes Resources Model and Kubernetes environment to provide an abstraction high-level & automation framework that automates clusters, logical clusters. The L-KaaS is a project standing on the Cluster API project. 
```

File: docs/design/diagrams/creating-logical-cluster-diagram.png
Comment (Jangwon-Lee-DCN) [inline docs/design/diagrams/creating-logical-cluster-diagram.png:1]: A logical cluster is created by EMCO using created clusters on Cloud Provider. So you have to add corresponding block and registration flow to EMCO.

File: docs/design/diagrams/functional.png
Comment (Jangwon-Lee-DCN) [inline docs/design/diagrams/functional.png:1]: A Logical cluster can be grouped with Private and Public cloud both. And you should describe the namespaces of the logical cluster. The logical cluster can use clusters for all resources or a specific namespace with a resource quota. Please check and follow EMCO’s DCM documents.

File: docs/design/diagrams/reference.png
Comment (Jangwon-Lee-DCN) [inline docs/design/diagrams/reference.png:1]: It’s same. Please change the Cloud Infrastructure block.

**Review by Jangwon-Lee-DCN — CHANGES_REQUESTED** [review]
I commented to some files.
IMO, please change design/README.md to explain everything roughly, not just “see something” and add embedded link for detailed like flows and operations. Maybe, a new file is also okay at desgin/something.

**Comment by ntnguyencse** [comment]
> I commented to some files. IMO, please change design/README.md to explain everything roughly, not just “see something” and add embedded link for detailed like flows and operations. Maybe, a new file is also okay at desgin/something.

I'll add introduction and explanation for each of it.

**Pull request closed without merging**